src/status_im2/contexts/onboarding/create_profile/view.cljs

```diff
@@ -194,11 +195,13 @@
             {:size   :paragraph-2
              :weight :medium
              :style  style/color-title}
-            (i18n/label :t/accent-colour)]
-           [quo/color-picker
-            {:blur?            true
-             :default-selected :blue
-             :on-change        on-change}]]]]]
+            (i18n/label :t/accent-colour)]]]
+         [quo/color-picker
+          {:blur?            true
+           :default-selected :blue
+           :on-change        on-change
+           :window-width     window-width
+           :container-style  {:padding-left (int (/ window-width 18.75))}}]]]
```

Comment:
where is this magic 18.75 coming from?

Reply:
I wanted to add padding 20 like the design, But I wanted to make it responsive.
18.75 = iPhone 11 Pro VW / 20

Reply:
Ok, well this should be in a def and then leave a comment explaining the value or some means to communicate this idea 👍

Reply:
Okay, Will do